cpp/models/abm/location.cpp

```diff
@@ -229,7 +229,7 @@ void Location::add_subpopulations_timepoint(const TimePoint& t)
     // Get the previous time point index. 
     // Since index starts from 0, we need to -1 from the current time point. 
     auto last_idx = m_subpopulations.get_num_time_points() - 1;
-    m_subpopulations.add_time_point(t.days());
+    m_subpopulations.add_time_point(t.hours());
```

Comment:
I am not sure you want to store it in hours. Do you? What do you think @DavidKerkmann ?

Reply:
In the simulation.cpp, the time step (m_dt) is set at hours(1). It is used in the 'advance' function where 'add_subpopulations_timepoint' got called. In this sense, the time series add a new timepoint per hour. 

If you think it is necessary to have time series in days, I could further add specific implementation for this.

Reply:
Yes, for now, one time point per hour is added. In general, the time step might change to something else, maybe even varying. It could also be 0.125749 days, for example. t.days() or t.hours() just gives you the double value that is saved in the time series. To be consistent with previous savings and the ODE, converting to t.days() is preferred.

Reply:
@khoanguyen-dev Please have a look how hours() and hours(int x) work: https://github.com/DLR-SC/memilio/blob/main/cpp/models/abm/time.cpp It is always internally saved in seconds.

Reply:
Yes! I have now convert the Timepoint to days.